scripts/run_app.sh

```diff
@@ -15,13 +15,15 @@ PLATFORM=""
 ENV_NAME="alfajoresdev"
 RELEASE=false
 SIMULATOR=""
+PROFILE=false
```

Comment:
nit: to clear any confusion
```suggestion
PROFILING_ENABLED=false
```

Reply:
Changed in 506f0d2d2df5fb282a352c3066fe9315b0a35f35!